index.rst

```diff
@@ -1,40 +1,378 @@
-  ^^^^^^^^^^^^^^
+In SQuaRE, the Science Quality and Reliability Engineering group, we realized that we often wanted to write stand-alone documents to describe our work.
+These documents didn't fit the conversational nature of `Community forum`_ topics, nor did they qualify for DM's change-controlled design document series, or fit into academic journals or the arXiv_.
+Traditionally, Confluence was meant to fill this niche, and indeed, the wikis are well used by DM.
+In practice, however, wikis have served DM poorly.
+Confluence pages are hard to find and discover.
+The editing and version control experience is also clumsy from the perspective of DM developers used to working on GitHub.
 
-  To add images, add the image file (png, svg or jpeg preferred) to the
-  _static/ directory. The reST syntax for adding the image is
+In the fall of 2015, we experimented with moving several DM design documents from Word documents stored on Docushare to documents that are written in `reStructuredText`_, built with `Sphinx`_, hosted on GitHub, and deployed to the web with `Read the Docs`_.
+The impetus behind the transition was that documents written in plain text and hosted on GitHub were far more likely to be kept up-to-date by DM developers.
 
-  .. figure:: /_static/filename.ext
-     :name: fig-label
-     :target: http://target.link/url
+Technotes were thus built on the same technology stack as the reStructuredText-based design documents, but re-purposed for general use by DM team members.\ [#]_ 
 
-     Caption text.
+.. [#] In fact, this technote applies equally to describing the new reStructuredText-based design document platform as it does the technote platform. The only meaningful difference is that design documents must be approved by a control board, and are ultimately deposited in LSST's Docushare.
 
-   Run: ``make html`` and ``open _build/html/index.html`` to preview your work.
-   See the README at https://github.com/lsst-sqre/lsst-report-bootstrap or
-   this repo's README for more info.
+Some of the possible applications for technotes are:
 
-   Feel free to delete this instructional comment.
+- to report the results of a project, such as a data processing or software development experiment,
+- to announce a new technology, serving as a high-level overview complementing software documentation,
+- to propose an architecture, possibly becoming the subject of a request for comment (RFC).
 
-Write the docs; right here.
+Technote Platform Design Choices
+=================================
+
+Although technotes are still in an early, minimally-viable state, (see :ref:`implementation`), they also demonstrate several design decisions that characterize the platform.
+
+Technotes are native to the web
+-------------------------------
+
+Although a preponderance of astronomy literature is published as PDFs, and many management documents as Word documents, those formats are archaic solutions from an era when documents were read in print.
```

Comment:
Drop "archaic". It's inflammatory and Word and PDF are perfectly good solutions for what they were designed for. How about: "those formats are optimized for print media"?